libd-llvm/src/d/llvm/expression.d

```diff
@@ -468,6 +468,7 @@ struct ExpressionGen {
 			auto thisPtr = LLVMBuildExtractValue(builder, dg, m.hasContext, "");
 			auto vtblPtr = LLVMBuildStructGEP(builder, thisPtr, 0, "");
 			auto vtbl = LLVMBuildLoad(builder, vtblPtr, "vtbl");
```

Comment:
It is incorrect to consider the function pointer as invariant as it is can be different in different instances if they are of different subclasses. Really, the value of this pointer is tied to the value of the vtable pointer.